README.pod

```diff
@@ -63,6 +63,27 @@ a subset of the functionality).
 It is based on Martin Berends' MiniDBI project, but unlike MiniDBI, DBDish
 aims to provide an interface that takes advantage of Perl 6 idioms
 
+=head2 Connecting to, and disconnecting from, a database
+
+For a database named I<hierarchy>, a connection may be established with any of
+the following syntaxes:
+
+    my ($dbh, $db);
+
+    $dbh = DBIish.connect("Pg", :database<hierarchy>);
+    $dbh.dispose;
+
+    $db = 'hierarchy';
+
+    $dbh = DBIish.connect("Pg", :database($db));
+    $dbh.dispose;
+
+    $dbh = DBIish.connect("Pg", database => $db);
+    $dbh.dispose;
+
+    $dbh = DBIish.connect("Pg", :$db);
```

Comment:
But none of my Postgres databases is called 'jkeenan'; the one in question is called 'hierarchy'.  Since there are several such DBs, I want to put the name into a variable.

Now, I should add that I included
```perl6
$dbh = DBIish.connect("Pg", :$db);
```
in the pull request not because I have any fondness for that syntax over the others, but simply because it works.  I don't know why it works; it just does.  If you think it should not work and feel that it should be made *not* to work, then we can avoid documenting it.  However, that's an exceptional case; in general, any way of calling a method that *does* work should be documented.

Thank you very much.

Jim Keenen